racetime/models/race.py

```diff
@@ -1956,5 +1963,16 @@ def override_stream(self, overridden_by):
         else:
             raise SafeException('Possible sync error. Refresh to continue.')
 
+    def update_split(self, split_name, split_time, is_finish):
+        split = {
+            'split_name': split_name,
+            'split_time': split_time,
+            'is_undo': split_time == '-',
+            'is_finish': is_finish,
+            'username': self.user.name
```

Comment:
The username was primarily used to create human-readable comparison names. user#descriminator should be unique, so would using user.full_name be a sufficient alternative?

Reply:
`full_name` would work. I'd recommend using `hashid` since it's never likely to change in format, and it's specifically designed for the task of uniquely identifying someone. You could send `hashid` for finding the correct entrant along with `name` for rendering the comparison.

There is also a slight edge case (which is *kinda* unlikely to cause any issue here) that a user may change their name/scrim when they're not actively racing (even though the race itself may still be ongoing).